modules/microshift-kickstart-prep.adoc

```diff
@@ -42,9 +42,9 @@ $ export PULL_SECRET="$(cat ~/.pull-secret.json)"
 * Set a password in the `PASSWD_TEXT` variable to use in the`USER_PASSWD` setting by running the following command:
 +
 .Example command setting the `PASSWD_TEXT` variable.
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ PASSWD_TEXT=<my_redhat_user_plain_text_password> <1>
+$ PASSWD_TEXT=_<my_redhat_user_plain_text_password>_ <1>
```

Comment:
certainly out of scope, but we don't typically use things like "my" or "your" in replaceable values. `<red_hat_user_plain_text_password>` seems right here to me